### PR TITLE
Updating docs

### PR DESCRIPTION
**Pull request by szemere**
The main goal of this PR is to add the preferred commit message format to the CONTRIBUTION.md description. (Currently it is only available in gitbook.)

Updated the dbld README files to represent the new manifest based package handling.

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -124,16 +123,14 @@ To contact us, visit the [mailing list][ar:mailing-list] where you can
...
 and patches. We are also using [Travis CI][ar:travis] for automatic
 testing.
 
- [ar:irc]: irc://chat.freenode.net/#syslog-ng
+ [ar:gitter]: https://gitter.im/balabit/syslog-ng
  [ar:mailing-list]: http://lists.balabit.com/mailman/listinfo/syslog-ng
  [ar:issue-tracker]: https://github.com/balabit/syslog-ng/issues
  [ar:issues:help]: https://github.com/balabit/syslog-ng/labels/help
- [ar:waffle]: https://waffle.io/balabit/syslog-ng
  [ar:travis]: https://travis-ci.org/algernon/syslog-ng/
```
Comment (mitzkia) [inline CONTRIBUTING.md]:
```suggestion
 [ar:travis]: https://travis-ci.org/balabit/syslog-ng/
```

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Pull request merged**